packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# 50.4.0 - November 5, 2020
```

Comment:
cc @ekapner

Reply:
@smaddox-sf - were you going to do some doc work based on retrieve mult components? Or do you need me to add it to the list?

Reply:
@ekapner - no not for this release.  at some point in the iteration, i want to update this page - https://developer.salesforce.com/tools/vscode/en/user-guide/perf-enhancements.  I don't expect to have time until Dec for the updates I mentioned